clang/include/clang/Basic/AllDiagnostics.h

```diff
@@ -14,6 +14,7 @@
 #ifndef LLVM_CLANG_BASIC_ALLDIAGNOSTICS_H
 #define LLVM_CLANG_BASIC_ALLDIAGNOSTICS_H
 
+// clang-format off
```

Comment:
What formatting is it doing here?  We very much LIKE our include file reordering/formatting, so I wonder what it is messing up here?  Can you share the format-diff for me?

Reply:
I see what is suggested here: https://github.com/llvm/llvm-project/actions/runs/17110487879/job/48530463722?pr=154618

What is wrong with its sugestion?  It looks right to me?